monai/auto3dseg/algo_gen.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from monai.transforms import Randomizable
+
+
+class Algo:
```

Comment:
if algo and algogen are independent classes, shall we separate them into two files?

Reply:
there are cases where we put the base classes in the same file, for example https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/transform.py. but I think it's just a style preference, I'm fine with either way. @Nic-Ma @mingxin-zheng